### ISSUE TITLE
problem narrowing down results of options

### ISSUE THREAD

**Issue by docteurklein** [NixOS/nixos-search#695]
Hello! I just discovered that ES is directly accessible publicly so I played a little with the query being sent out, 
and found out we explode each search word by space to become its own `wildcard` query in a `must` block, possibly capping the selectivity of the query, thereby violating the readme

**Comment by Dessix** [comment]
So far as I can tell, there's no `&&` syntax equivalent; Adding new items to the search always gives the results of both separate searches, and makes nix-search effectively useless compared to `grep` or even just browsing the github repo. Am I missing something, or is this the extent of the issue?

**Comment by docteurklein** [comment]
Yes I think that's basically my issue. I'dl like if we could have some kind of `&&` operator to narrow things down.
